draft-ietf-tls-subcerts.md

```diff
@@ -136,20 +140,21 @@ draft-02
 A delegated credential is a digitally signed data structure with two semantic
 fields: a validity interval and a public key (along with its associated
 signature algorithm).  The signature on the credential indicates a delegation
-from the certificate that is issued to the TLS server operator.  The secret key
-used to sign a credential corresponds to the public key of the TLS server's
+from the certificate that is issued to the peer.  The secret key
+used to sign a credential corresponds to the public key of the peer's
 X.509 end-entity certificate.
 
 A TLS handshake that uses delegated credentials differs from a normal handshake
 in a few important ways:
 
-* The client provides an extension in its ClientHello that indicates support
-  for this mechanism.
-* The server provides both the certificate chain terminating in its certificate
-  as well as the delegated credential.
-* The client uses information in the server's certificate to verify the
-  delegated credential and that the server is asserting an expected identity.
-* The client uses the public key in the credential as the server's
+* The initiating peer provides an extension in its ClientHello or CertificateRequest
+  that indicates support for this mechanism.
+* The peer sending the Certificate message provides both the certificate
+  chain terminating in its certificate as well as the delegated credential.
+* The authenticating intitiator uses information from the peer's certificate
+  to verify the delegated credential and that the peer is asserting an
+  expected identity.
+* Both peers use the public key in the credential as the certificate's
```

Comment:
maybe here say 

Peers accepting the delegated credential use it as the certificate's working key for the TLS hadshake